navi/src/test/java/com/trello/navi2/rx/RxNaviActivityTest.java

```diff
@@ -13,15 +13,15 @@
 import io.reactivex.observers.TestObserver;
 import org.junit.Test;
 
+import static com.trello.navi2.rx.RxNavi.observe;
```

Comment:
Hah good catch, no, that must've been a slip on the auto-complete when I was doing stuff.